Makefile

```diff
@@ -1,7 +1,7 @@
-IMAGE_NAME = vtex/checkout-ui-healthckeck-$(ENVIRONMENT)
+IMAGE_NAME = vtex/checkout-ui-healthcheck-$(ENVIRONMENT)
 
-deamon: build
-	docker run --ipc=host --shm-size 1024M -d $(IMAGE_NAME)
+daemon: build
+	docker run --ipc=host -it --shm-size 1024M $(IMAGE_NAME) yarn test:$(ENVIRONMENT)
```

Comment:
does this still run in the background?

Reply:
No. Should it? I really don't know if this makes a difference when running in CI, and when running locally I'd like to see the output of the tests in the console.

Reply:
hmmm makes sense, can we rename the script from `daemon` to `run` then?

Reply:
Sure, done.